llvm/test/CodeGen/SPIRV/pointers/store-operand-ptr-to-struct.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s
```

Comment:
In case of this test, the generated SPIR-V appears to be working fine with the NEO driver but differs when compared to the output of SPIR-V Translator. OpFunctionParameter has i32* type instead of %nested_struct*.